ait/gui/static/js/ait/gui/Plot.js

```diff
@@ -78,21 +78,21 @@ class DygraphsBackend
         }
     }
 
-    plot (packet) {
-        const pname = packet._defn.name
+    plot (data) {
+        const pname = data['packet']
+        let packet = data['data']['raw']
```

Comment:
Is this change working as expected with DN-to-EU'ed and raw fields? We'll need to double check and make sure it's working the same. Not saying it's broken, I just would have expected more abstraction when grabbing the raw vs DN-to-EU field value

Reply:
This plots the raw fields only. What is the expected functionality, that it plots dn-to-eus if they're available?